addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -304,6 +314,41 @@ def test_systray_get_activities(self):
             }
         )
 
+    def test_multi_company_notification(self):
+        # Disable the multi-company rule to collaborate between companies
+        self.env.ref('test_mail.mail_test_ticket_mc_rule').action_archive()
```

Comment:
@tde-banana-odoo Hello, the test seems fine by me, but I have no clue if removing this rule is reasonable or not, and if we should adapt our code considering this could be disabled, kindly asking your opinion here :)